src/components/text/index.tsx

```diff
@@ -192,14 +191,13 @@ const styles = StyleSheet.create({
   container: {
     backgroundColor: 'transparent',
     color: Colors.$textDefault,
-    ...Platform.select({
-      ios: {
+    ...(Constants.isIOS
```

Comment:
Nice idea!
Did you verify this change works (in the app, not tests)

Reply:
Seems to be working